css/main.css

```diff
@@ -290,17 +291,26 @@ table th {
 	opacity:0.3;
 }
 
-.mod1 { background-color:#97b911; color:#000000; }
-.mod2 { background-color:#009ee0 ; color:#FFFFFF; }
-.mod3 { background-color:#007930; color:#FFFFFF; }
-.mod4 { background-color:#a7d6c9; color:#000000; }
-.mod5 { background-color:#cd071e; color:#FFFFFF; }
-.mod6 { background-color:#ffed00; color:#000000; }
-.mod7 { background-color:#e06c9e; color:#FFFFFF; }
-.mod8 { background-color:#f7a800; color:#000000; }
-.mod9 { background-color:#972f86; color:#FFFFFF; }
-
-
+.mod1 { background-color:#97b911!important; color:#000000!important; }
```

Comment:
Try the selector ".section .modx"; it's probably not changing because the section it's in also specifies a color, and this selector isn't more specific.